### PR TITLE
feat(auth-pages): allow translation for required input field messages

### LINKED ISSUE #5855: [FEAT] Antd | Allow translation for required field messages
### Is your feature request related to a problem? Please describe.

Looks like the rules for form validation in the auth page component's do not allow for setting translated messages to the auth components. This is also true for the register, forgot password, etc. pages.

```
<Form.Item
            name="email"
            label={translate("pages.login.fields.email", "Email")}
            rules={[
              { required: true },
              {
                type: "email",
                message: translate(
                  "pages.login.errors.validEmail",
                  "Invalid email address",
                ),
              },
            ]}
          >
```

Screenshot 2024-04-15 at 5 27 32 PM

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Describe the thing to improve

From the looks of Antd's documentation, you can also pass a message property to the "required" rule:

https://ant.design/components/form#components-form-demo-normal-login

### PR DESCRIPTION
**Pull request by Ac-Srikanth**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://refine.dev/docs/guides-concepts/contributing/#commit-convention

## Bugs / Features

- [x] Related issue(s) linked
- [ ] Tests for the changes have been added
- [x] Docs have been added / updated
- [x] Changesets have been added https://refine.dev/docs/guides-concepts/contributing/#creating-a-changeset

## What is the current behaviour?
Auth Input fields which were required  had no custom error message feature.

## What is the new behaviour?
can provide Custom error message  for required Auth Input fields with correct translation.

fixes # (issue)
#[5855](https://github.com/refinedev/refine/issues/5855)

## Notes for reviewers
This is my first PR , please do suggest changes or more things to be added if i missed out any.

### CODE REVIEW

**Review by aliemir — CHANGES_REQUESTED** [review]
Hey @Ac-Srikanth thank you for the PR! We'll review it soon. Before we do, can you please check [Creating a changeset](https://refine.dev/docs/guides-concepts/contributing/#creating-a-changeset) section in our contributing guide? We'll need a changeset for the related changes 💯

**Comment by aliemir** [comment]
Hey @Ac-Srikanth thank you for the update! I was reviewing the PR and just noticed that we also require a small update on our live previews package to reflect i18n changes;

Can you provide the keys you've created to the files below:

- `packages/live-previews/public/locales/en/common.json`
- `packages/live-previews/public/locales/de/common.json`

We've marked this PR to be included in our May release and I'm hoping to complete its review if you can provide the changes I mentioned above 🙏

**Comment by Ac-Srikanth** [comment]
hey @aliemir thank you for the review , i will add the keys to these files as well and yes this can be done before the may release. Very soon you can see the changes.

File: .changeset/cold-emus-visit.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+"@refinedev/antd": minor
+"@refinedev/chakra-ui": minor
+"@refinedev/mui": minor
+"@refinedev/live-previews": minor
```
Comment (aliemir) [inline .changeset/cold-emus-visit.md:5]: live-previews is not published so we're not including it in our changesets. We can safely remove it.
```suggestion
```

**Review by aliemir — APPROVED** [review]
Left a small change request in the changeset, other than that I think we're good to go! Thank you for your contribution! 🚀

**Review by aliemir — APPROVED** [review]

**Pull request merged**